code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -234,6 +234,7 @@
 	throw_range = 3
 	hitsound = 'sound/items/trayhit1.ogg'
 	hit_reaction_chance = 50
+	materials = list(MAT_METAL = 500)
```

Comment:
but chairs are made of 2000 metal? knocking a chair over doesn't reduce the amount of metal in it.